### PR TITLE
Ignore permission errors in count_rasa_temp_files

### PR DESCRIPTION
**Pull request by federicotdn**
**Proposed changes**:
The function `count_rasa_temp_files` can fail to run if it encounters a directory it can't access. Ignore those cases as they aren't needed for the tests to run.

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by erohmensing** [comment]
So is it essentially, if we can't count them, we also can't delete them, so the number counted and deleted will always be the same?

**Comment by federicotdn** [comment]
@eliseboyd I see it as: if we can't access directory xyz, then it probably means that it wasn't created by us in the first place.

**Comment by erohmensing** [comment]
Ah, true. Just making sure the numbers will all turn out alright in the end in the tests -- but Travis will tell us that. PS, you keep tagging the wrong person when you try to tag me 😅

**Comment by federicotdn** [comment]
@erohmensing True!! Hadn't noticed

File: tests/cli/test_rasa_train.py [hunk 1]
```diff
@@ -124,9 +124,13 @@ def count_rasa_temp_files():
         if not entry.is_dir():
             continue
 
-        for f in os.listdir(entry.path):
-            if f.endswith("_nlu.md") or f.endswith("_stories.md"):
-                count += 1
+        try:
+            for f in os.listdir(entry.path):
+                if f.endswith("_nlu.md") or f.endswith("_stories.md"):
```
Comment (erohmensing) [inline tests/cli/test_rasa_train.py]: why are these preceded with underscores? don't they usually come out as just stories.md or nlu.md?

Comment (federicotdn) [reply tests/cli/test_rasa_train.py]: I believe both options would work. `_copy_files_to_new_dir` was creating the files using:
```python
unique_file_name = unique_prefix + "_" + os.path.basename(f)
```

**Pull request merged**